### DISCUSSION TITLE
Workflow Execution UI - Running In-Memory Server

### DISCUSSION THREAD

**Discussion by krishnapyde** [Netflix/conductor#2780] [Q&A] [answered]
I am using Conductor as In-Memory Server.
After executing a workflow, I am unable to see that workflow on the 'Workflow Execution UI'.
I am able to see the Task and Workflow definitions I created. But the executions are not getting displayed.
Is this a limitation in the In-Memory Server? Or am I missing something? Can you help me on this?
Note: 
I am running the application on my laptop
I have stared the UI using -> yarn run start
We will eventually move to the production version, but for now, I am doing a POC to get a hang of the conductor.

![image](https://user-images.githubusercontent.com/20985617/155267000-ac048bb7-8bce-4bf6-b380-f2869546269d.png)

**Comment by v1r3n** [comment, marked as answer]
Hi,

UI relies on Elasticsearch for all the queries, so when you run Conductor server in memory, there is no ES and UI does not show any workflows.

You can however, visualize the flow using URL: `http://localhost:5000/execution/<workflow_id>`

**Reply by krishnapyde** [reply]
Excellent!
Thanks, @v1r3n . This helps.

**Reply by krishnapyde** [reply]
Can this information be added back to the Conductor Documentation page? I am sure there will be others stumbling upon this at some point in time.
